lms/djangoapps/courseware/tests/test_masquerade.py

```diff
@@ -149,6 +149,18 @@ def verify_show_answer_present(self, show_answer_expected):
         self.assertIn(self.problem_display_name, problem_html)
         self.assertEqual(show_answer_expected, "Show Answer" in problem_html)
 
+    def verify_original_user_profile_link(self):
```

Comment:
Nit: Elsewhere, we use the term "real user" for this role.  Maybe use the same name here, to help emphasize that terminology.